dot/network/discovery.go

```diff
@@ -115,6 +121,9 @@ func (d *discovery) stop() error {
 		return nil
 	}
 
+	ethmetrics.Unregister(checkPeerCountMetrics)
+	ethmetrics.Unregister(peersStoreMetrics)
```

Comment:
what does this do?

Reply:
this allows the unused metrics to be collected by the gc